WordPress/Classes/ViewRelated/Themes/ThemeBrowserCell.swift

```diff
@@ -193,11 +193,34 @@ open class ThemeBrowserCell: UICollectionViewCell {
         activityView.stopAnimating()
     }
 
+    private func imageUrlForWidth(imageUrl: String) -> String {
+        var screenshotUrl = imageUrl
+        // Themes not hosted on WP.com have an incorrect screenshotUrl
+        // it uses a // url (this is used on the web) when the scheme is not known.
+        if !screenshotUrl.hasPrefix("http") && screenshotUrl.hasPrefix("//") {
+            // Since not all sites support https
+            screenshotUrl = String(format: "http:%@", imageUrl)
+        }
+
+        guard var components = URLComponents(string: screenshotUrl) else {
+            return screenshotUrl
+        }
+
+        var queryItems: [URLQueryItem] = components.queryItems ??  []
+        queryItems.append(URLQueryItem(name: "w", value: "\(presenter!.screenshotWidth)"))
```

Comment:
I [think](https://github.com/wordpress-mobile/WordPress-iOS/pull/16914#discussion_r675602910) we should handle this `presenter!` by conditionally unwrapping. 

I can push a change here though since I know you're away this week.